src/main/java/teammates/ui/webapi/action/JsonResult.java

```diff
@@ -46,6 +57,13 @@ public void send(HttpServletResponse resp) throws IOException {
         resp.setContentType("application/json");
         PrintWriter pw = resp.getWriter();
         pw.print(JsonUtils.toJson(output));
+        for (Cookie cookie : cookies) {
```

Comment:
Just a minor issue.

The sending of cookie should before the sending of body. In HTTP, cookie is before the body. Not sure the wrong order will cause problems or not.